expression/builtin_json_vec.go

```diff
@@ -792,11 +793,73 @@ func (b *builtinJSONMergeSig) vecEvalJSON(input *chunk.Chunk, result *chunk.Colu
 }
 
 func (b *builtinJSONContainsPathSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinJSONContainsPathSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	jsonBuf, err := b.bufAllocator.get(types.ETJson, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(jsonBuf)
+	if err := b.args[0].VecEvalJSON(b.ctx, input, jsonBuf); err != nil {
+		return err
+	}
+	typeBuf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(typeBuf)
+	if err := b.args[1].VecEvalString(b.ctx, input, typeBuf); err != nil {
+		return err
+	}
+	pathBufs := make([]*chunk.Column, len(b.args)-2)
+	defer func() {
+		for i := 0; i < len(pathBufs); i++ {
+			if pathBufs[i] != nil {
+				b.bufAllocator.put(pathBufs[i])
+			}
+		}
+	}()
+	for i := 0; i < len(pathBufs); i++ {
+		pathBuf, err := b.bufAllocator.get(types.ETString, n)
+		if err != nil {
+			return err
+		}
+		if err := b.args[2+i].VecEvalString(b.ctx, input, pathBuf); err != nil {
+			return err
+		}
+		pathBufs[i] = pathBuf
+	}
+
+	result.ResizeInt64(n, false)
+	result.MergeNulls(jsonBuf, typeBuf)
+	i64s := result.Int64s()
+	for i := 0; i < n; i++ {
+		containType := strings.ToLower(typeBuf.GetString(i))
```

Comment:
https://github.com/pingcap/tidb/blob/master/expression/builtin_json.go#L592-L594
This check is lost

Reply:
Thank you, fixed.